plio/cache.py

```diff
@@ -0,0 +1,34 @@
+from django.core.cache import cache
+
+
+def get_cache_key(instance):
+    """Calculates cache key for an instance"""
+    instance_class = instance.__class__.__name__
+    return {
+        "Video": f"video_{instance.pk}",
+        "Plio": f"plio_{instance.pk}",
+        "Session": f"session_{instance.pk}",
+        "User": f"user_{instance.pk}",
+        "UserMeta": f"user_meta_{instance.pk}",
+    }.get(instance_class, None)
```

Comment:
this needs to include question, image and item too as well right?

Why are we including session here? as in, unlike a plio, once a session has been created, changing anything in the plio won't change anything about the session right?

Reply:
Yes, questions and items I'll add.

We are currently sending plio details in the session response (along with a lot of other information). Since we are not making any GET requests for the session, so unlike Plio response, I don't think the whole session response will be cached. So we don't need session cache key here. I'll remove.

But we can do it in this way. Session response includes:
1. Plio (can come from cache)
2. User (can come from cache)
3. Experiment (can come from cache)
4. Last Event (won't be cached)
5. Session Answers (probably won't be cached)